packages/clarity-js/src/layout/node.ts

```diff
@@ -108,6 +108,11 @@ export default function (node: Node, source: Source): Node {
                     }
                     break;
                 case "NOSCRIPT":
+                    // keeping the noscript tag but removing all of its contents. Some HTML markup relies on having these tags
+                    // to maintain parity with the original css view, but we don't want to execute any noscript in Clarity
```

Comment:
>maintain [](http://example.com/codeflow?start=26&length=8)

Just asking for us to think through: Why not send the content to avoid even more bizarre cases of CSS that refer to noscript tag's content?

Reply:
That's not possible (to the best of my knowledge anyway). Sending the content is wasteful anyway as javascript will always be enabled in playbacks (otherwise they won't play :P )